fleetspeak/src/server/servertests/comms_test.go

```diff
@@ -289,11 +289,9 @@ func TestStoredMessagesFromBlocklistedClient(t *testing.T) {
 	defer fin()
 
 	ctx := context.Background()
-	var blocklistedCount uint = 0
-	var nonBlocklistedCount uint = 0
-	testService := blocklistService{&blocklistedCount, &nonBlocklistedCount}
+	testService := blocklistService{}
```

Comment:
Similar to prev. comment: you intend to use this as a pointer, so you can initialize this as `&blocklistService{}` directly, and avoid the `&` below

Reply:
Done.